builder/proxmox/iso/config_test.go

```diff
@@ -109,6 +110,9 @@ func TestBasicExampleFromDocsIsValid(t *testing.T) {
 	if b.config.CloudInit != false {
 		t.Errorf("Expected CloudInit to be false, got %t", b.config.CloudInit)
 	}
+	if b.config.SkipConvertToTemplate.False() {
+		t.Errorf("Expected ConvertToTemplate to be true, got %t", b.config.SkipConvertToTemplate.True())
```

Comment:
This should be rephrased, also I suspect the error was introduced in the first draft, so the value does not match what the condition highlights.